docs/source/installation.md

```diff
@@ -0,0 +1,26 @@
+# Installation
```

Comment:
This seems to duplicate content currently in readme.md - perhaps readme.md should be changed to link to this content so install instructions are all in one place?

Reply:
> This seems to duplicate content currently in readme.md - perhaps readme.md should be changed to link to this content so install instructions are all in one place?

Great suggestion. I will keep a simple version in Readme and provide a link for more details.